openedx/features/course_experience/__init__.py

```diff
@@ -109,7 +104,7 @@ def default_course_url_name(course_id):
     return 'openedx.course_experience.course_home'
 
 
-def course_home_url_name(course_key):
+def course_home_url_name(course_key):  # pylint: disable=unused-argument
```

Comment:
is this function needed anymore? (no)

Reply:
Nah, I would remove it. Especially now that it doesn't depend on `course_key`